### ISSUE TITLE
Social information displayed in wrong slots

### ISSUE THREAD

**Issue by mikaere66** [christinloehner/linuxcounter.new#71]
In my case, I only have Facebook and Google+ but FB when displayed goes into the MSN slot, and G+ goes into the FB slot
![screenshot from 2016-05-08 20 47 47](https://cloud.githubusercontent.com/assets/3999272/15096877/50b16768-155e-11e6-8d13-ff5cb38c4d17.png)

**Comment by corrafig** [comment]
Same issue here. Twitter info ends up to Google+ information in the profile page.

**Comment by tallship** [comment]
Yes @mikaere66 , this issue #68 was addressed and just closed. Like @Nikolice mentioned, he applied a patch and I think you'll be pleasantly surprised once you login and check :)
